docs/xenserver-xencert-guide.md

```diff
@@ -331,13 +331,17 @@ Execution time: 47 minutes, 42 seconds.
 ### Appendix A-Blocking paths for failover testing  
 
 ##### iSCSI storage type  
-For iSCSI storage type, the paths can be failed over by using the iptables utility:  
+For iSCSI storage type, the paths can be failed over by using the nftables:  
 
-    iptables –A OUTPUT –d <IP address> -j DROP or iptables –A INPUT –s <IP address> -j DROP  
+    nft add table inet filter

```

Comment:
This probably won't work once the firewalld migration is complete as we no longer build the compatibility bits into `nft`. It is for this reason that XenRT has been updated to use OVS rules when performing equivalent tests, see - https://code.citrite.net/projects/XRT/repos/xenrt/commits/f214e113c2b1b57aea481898574406beeaa96084#exec%2Fxenrt%2Flib%2Fxenserver%2Fhost.py